README.md

```diff
@@ -3,21 +3,25 @@ Facebook Thrift [![Build Status](https://travis-ci.org/facebook/fbthrift.svg?bra
 
 Thrift is a serialization and RPC framework for service communication. Thrift enables these features in all major languages, and there is strong support for C++, Python, Hack, and Java. Most services at Facebook are written using Thrift for RPC, and some storage systems use Thrift for serializing records on disk.
 
-At a high level, Thrift is three major things:
+Facebook Thrift is not a distribution of [Apache Thrift](https://thrift.apache.org/). This is an evolved internal branch of Thrift that Facebook re-released to open source community in February 2014.  
+Although not all Apache Thrift changes are reflected in fbthrift, we track the upstream closely and hope to work with the maintainers to incorporate this work. Read more about what lead to this duality in the Facebook Code [blog post](https://code.facebook.com/posts/1468950976659943/under-the-hood-building-and-open-sourcing-fbthrift/)
```

Comment:
Note that Facebook Thrift has been diverging substantially from Apache Thrift in recent years for some target languages, especially C++, and will continue diverging, with new transport layers, support libraries, code gen, etc. So this paragraph is a bit out of place.

Reply:
I am not a user or a contributor yet, so I wasn't aware. Sorry about that. I'd still like to do whatever possible to make sure the main Readme file is clear about what Facebook Thrift is and what it __isn't__. This PR is about making sure that people won't be confused because of the naming and other shared characteristics.

What's a good way we could do that?

Reply:
Can say that Facebook Thrift was originally released closely tracking Apache Thrift but is now evolving in new directions.

Reply:
Will do.

If someone would like to learn more about these differences and follow the development — where would they go?